ui-tests/cypress/support/pageObjects/app/AppPage.js

```diff
@@ -53,6 +53,12 @@ export default class AppPage extends WebElement {
     return this.content;
   }
 
+  getModalDialogByTitle(title) {
```

Comment:
What is this needed for? Do you have multiple dialog at the same time?
There's already a getDialog method; if it is behaving differently for a specific page you could just override it for that page

Reply:
highlighted method has been removed. instead used `getDialog` method